docs/scripting.md

```diff
@@ -354,7 +354,23 @@ with open(outfile, "w") as out:
 
 ```
 
+### Basic Validation
+Some simple validation checks can be run using a couple of lambda functions to select and evaluate elements.
 
+```
+# As before assume we have a factory service and the model is loaded
+# Define a lambda to select an element
+element_select =  lambda e:isinstance(e, SysML.sysml.SysMLDiagram) and e.diagramType=='ibd'
```

Comment:
Hi @mikekidner, we have a lint rule to avoid assigning a lambda to a variable. An explanation of why is her: https://docs.astral.sh/ruff/rules/lambda-assignment/

I know that it is more concise, but it would probably be more readable if you define element_select and rule as full functions.